recipes/curl/curl.inc

```diff
@@ -24,22 +24,30 @@ EXTRA_OECONF_IPV6:USE_curl_ipv6 = "--enable-ipv6"
 
 RECIPE_FLAGS += "curl_ssl"
 DEPENDS:>USE_curl_ssl = " libssl libcrypto"
+RDEPENDS_${PN}:>USE_curl_ssl = " libssl libcrypto"
+RDEPENDS_${PN}-dev:>USE_curl_ssl = " libssl-dev libcrypto-dev"
```

Comment:
I just tested compilation of curl with a new Linux SDK with all RDEPENDS-dev removed and it works just fine.
I will modify the pull request